### PR TITLE
Small Floodfill Change

### PR DESCRIPTION
**Pull request by Aadit-Ambadkar**

### CODE REVIEW

**Comment by nathangong** [comment]
Since floodfill can also be done using BFS, I think it might be good to explicitly mention that this recursive DFS method is just one algorithm that can be used to perform floodfill.

**Comment by Aadit-Ambadkar** [comment]
Yeah, edited that in another place where it made more sense.

File: content/3_Silver/Flood_Fill.mdx [hunk 1]
```diff
@@ -554,7 +556,7 @@ Recursive implementations of flood fill sometimes lead to
 
 </Warning>
 
-A non-recursive implementation of flood fill follows:
+A non-recursive implementation of flood fill adds the edgesto a stack or queue, similar to [BFS](https://usaco.guide/gold/bfs), and is usually implemented as follows:
```
Comment (SansPapyrus683) [inline content/3_Silver/Flood_Fill.mdx:559]: > edgesto

typo?

File: content/3_Silver/Flood_Fill.mdx [hunk 2]
```diff
@@ -371,7 +371,9 @@ size of the current component we are visiting. Make sure to store the grid, the
 visited array, dimensions, and the current size variable globally.
 
 This means that we want to recursively call the search function for the squares
-above, below, and to the left and right of our current square. The algorithm to
+above, below, and to the left and right of our current square. Due to it's
```
Comment (nathangong) [inline content/3_Silver/Flood_Fill.mdx:374]:
```suggestion
above, below, and to the left and right of our current square. Due to its
```

File: content/3_Silver/Flood_Fill.mdx [hunk 3]
```diff
@@ -371,7 +371,9 @@ size of the current component we are visiting. Make sure to store the grid, the
...
+recursive nature, you can think of floodfill as a modified version of
```
Comment (nathangong) [inline content/3_Silver/Flood_Fill.mdx:375]: Second person isn't used at all in the explanation, so for consistency:
```suggestion
recursive nature, floodfill can be thought of as a modified version of
```

File: content/3_Silver/Flood_Fill.mdx [hunk 4]
```diff
@@ -554,7 +556,7 @@ Recursive implementations of flood fill sometimes lead to
 
 </Warning>
 
-A non-recursive implementation of flood fill follows:
+A non-recursive implementation of flood fill adds the edges to a stack or queue, similar to [BFS](https://usaco.guide/gold/bfs), and is usually implemented as follows:
```
Comment (nathangong) [inline content/3_Silver/Flood_Fill.mdx:559]: I'm pretty sure you add nodes, not edges to the stack/queue in floodfill. This is also the case in the code below.
```suggestion
A non-recursive implementation of flood fill adds adjacent nodes to a stack or queue, similar to [BFS](https://usaco.guide/gold/bfs), and is usually implemented as follows:
```

**Review by nathangong — CHANGES_REQUESTED** [review]
Just some small grammar/wording stuff

**Review by nwatx — APPROVED** [review]

**Review by nathangong — APPROVED** [review]

**Review by HarryWangATX — APPROVED** [review]

**Pull request merged**